poradnia/stats/utils.py

```diff
@@ -14,6 +14,16 @@ def raise_unless_unauthenticated(view, request):
         return redirect('/konta/login/?next=%s' % request.path)
     return None
 
+def split_while(seq, key):
```

Comment:
Tak, miałem na myśli właśnie połączoną funkcjonalność tych dwóch metod. Nie spodziewałem się ich znaleźć w itertools. Skoro są dostępne, to myślę, że po prostu z nich skorzystam i pozbędę się `split_while`.